docs/install/linux.md

```diff
@@ -212,24 +218,30 @@ hide:
     $ sudo dnf install -y {{ fedora39_rpm_stable }}
     ```
 
-=== "SUSE"
-    ## SUSE Linux
+=== "openSUSE"
+    ## openSUSE
 
     !!! note
-        It is recommended that you install via Copr so that it is easiest
-        to stay up to date as future versions of wezterm are released.
+        If you want nightly versions of WezTerm, it is recommended to use Copr.
```

Comment:
I reverted my changes. Since you are the maintainer, I rely on your expertise here. However, when someone reads this note, it may imply to always use the nightly builds instead of the stable releases.
If you are interested in what openSUSE maintainers are doing/modifying in WezTerm on openSUSE, take a look at the [packages in the OBS](https://build.opensuse.org/package/show/openSUSE:Factory/wezterm) or directly at the `.spec` [file](https://build.opensuse.org/projects/openSUSE:Factory/packages/wezterm/files/wezterm.spec). The interesting part begins at line 105.